crates/pallet-humanode-offences/src/lib.rs

```diff
@@ -72,38 +72,41 @@ where
             offenders: offenders.clone(),
         });
 
-        let mut should_be_deauthenticated = Vec::with_capacity(offenders.len());
+        let mut maybe_should_be_deauthenticated = Vec::with_capacity(offenders.len());
```

Comment:
Why `maybe_`?

Reply:
Because the old authentication can be passed still as a part of offenders data

Reply:
I see your point, will rename.

Reply:
https://github.com/humanode-network/humanode/pull/1480/commits/39eb549358bb3912d3b8ef0a2ddada56c5c0e7f3